common/reviews/api/rush-lib.api.md

```diff
@@ -7,6 +7,7 @@
 /// <reference types="node" />
 
 import { AsyncSeriesHook } from 'tapable';
+import { HookMap } from 'tapable';
```

Comment:
```suggestion
import { AsyncSeriesHook, HookMap } from 'tapable';
```

Reply:
Generated file, it'll get automatically revered next time it builds.

Reply:
Ah, didn't realize what file it was on.